### PR TITLE
update to instructions for local development exercise

### PR DESCRIPTION
**Pull request by meher-liatrio**
updates exercise instructions to  include a step to add a post-processor to upload box to cloud account, and a hint for installing vagrant-vmware-desktop.

### CODE REVIEW

File: docs/3-virtual-machines-containers/3.2-local-development.md [hunk 1]
```diff
@@ -48,10 +48,12 @@ Read [Creating a Base Box](https://www.vagrantup.com/docs/boxes/base) and [Creat
     ?> Many of Vagrant requirements are already taken care of for you in the example. It is up to you to figure out which one(s) you need to add.
 
 2. Add a [Vagrant `post-processor`](https://www.packer.io/docs/post-processors/vagrant/vagrant) to your Packer build template. Configure it to output both a VMWare Fusion image and a Vagrant box.
+- Add a vagrant post processor to automatically upload your box to your cloud acount
 
 3. Create a VagrantCloud account if needed and upload your box.
 
 4. Read and install required pieces for the [Vagrant VMWare provider](https://developer.hashicorp.com/vagrant/docs/providers/vmware)
+- Hint: use brew to install vagrant-vmware-desktop
```
Comment (jburns24) [inline docs/3-virtual-machines-containers/3.2-local-development.md:56]:
```suggestion
- Hint: For VMWare Utility use `brew install --cask vagrant-vmware-utility`
```

File: docs/3-virtual-machines-containers/3.2-local-development.md [hunk 2]
```diff
@@ -48,10 +48,12 @@ Read [Creating a Base Box](https://www.vagrantup.com/docs/boxes/base) and [Creat
     ?> Many of Vagrant requirements are already taken care of for you in the example. It is up to you to figure out which one(s) you need to add.
 
 2. Add a [Vagrant `post-processor`](https://www.packer.io/docs/post-processors/vagrant/vagrant) to your Packer build template. Configure it to output both a VMWare Fusion image and a Vagrant box.
+- Add a vagrant post processor to automatically upload your box to your cloud acount
```
Comment (jburns24) [inline docs/3-virtual-machines-containers/3.2-local-development.md:51]:
```suggestion
- Add a vagrant post processor to automatically upload your box to your cloud acount
```

Seems odd that this comes before step 3 (create a VagrantCloud account) can we move this towards the end of this exercise and maybe mark it Extra Credit (see [this](https://devops-bootcamp.liatr.io/#/3-virtual-machines-containers/3.3-managing-infrastructure?id=extra-credit) for an example)

Comment (meher-liatrio) [reply docs/3-virtual-machines-containers/3.2-local-development.md]: I have moved this to extra credit as suggested.

File: docs/3-virtual-machines-containers/3.2-local-development.md [hunk 3]
```diff
@@ -52,13 +52,18 @@ Read [Creating a Base Box](https://www.vagrantup.com/docs/boxes/base) and [Creat
...
 
 5. Create a Vagrant file that will pull your base box from VagrantCloud
 
 6. Run `vagrant up` and verify Vagrant can pull and start your VM.
 
 7. Verify that `vagrant ssh` will ssh into your VM.
 
+## Extra Credit
+
+ Add a post processor to automatically upload your vagrant box to Vagrant Cloud.
```
Comment (jburns24) [inline docs/3-virtual-machines-containers/3.2-local-development.md:65]:
```suggestion
 Add a post processor to automatically upload your packer build artifact to Vagrant Cloud.
```

**Review by jburns24 — APPROVED** [review]

**Pull request merged**